boards/arm/nucleo_f413zh/doc/nucleof413zh.rst

```diff
@@ -147,6 +151,11 @@ Serial Port
 Nucleo F413ZH board has 10 UARTs. The Zephyr console output is assigned to UART3.
 Default settings are 115200 8N1.
 
+USB
+===
+Nucleo F413ZH board has a USB 2.0 full-speed device interface available through
+its micro USB connector (USB USER).
```

Comment:
```
"The USB OTG is a dual-role device (DRD) controller that supports both device and host functions and is fully compliant with the On-The-Go Supplement to the USB 2.0 Specification. It can also be configured as a host-only or device-only controller, fully compliant with the USB 2.0 Specification."
```
According to the reference manual the board has a USB OTG device controller. I think, it would be better to mention here that it has a USB OTG controller and only USB device interface is supported for now.

Reply:
+1 for that technical addition too :)